internal/cluster/statemachine.go

```diff
@@ -280,6 +271,17 @@ func NewClusterStateMachine(th *transitionHandler) stateswitch.StateMachine {
 		PostTransition:   th.PostRefreshCluster(statusInfoError),
 	})
 
+	for _, state := range []stateswitch.State{
+		stateswitch.State(models.ClusterStatusInstalling),
+		stateswitch.State(models.ClusterStatusFinalizing)} {
+		sm.AddTransition(stateswitch.TransitionRule{
+			TransitionType:   TransitionTypeUpdateInstallationProgress,
+			SourceStates:     []stateswitch.State{state},
```

Comment:
you don't need a for, you the parameter is already a list so just assign it directly.

Reply:
I did this to match the `SourceState` with `DestinationState`

Reply:
oh right